app/src/main/java/org/keepgoeat/util/extension/StringExt.kt

```diff
@@ -0,0 +1,13 @@
+package org.keepgoeat.util.extension
+
+import java.text.BreakIterator
+
+fun String?.getStringLength(): Int {
+    val it: BreakIterator = BreakIterator.getCharacterInstance()
```

Comment:
```suggestion
    if (this == null) return 0
    val it: BreakIterator = BreakIterator.getCharacterInstance()
```
빠르게 화면 진입했을 때 목표 음식(goalFood: String?)이 초기화 되지 않은 경우 null일 것이고, 7번라인에서 setText할 때 NPE 발생할 것입니다! 널처리 해주는 것이 좋아보이네용~

Reply:
넹! 널처리 적용하도록 하겠습니당~~